lib/ansible/modules/system/authorized_key.py

```diff
@@ -227,6 +227,7 @@
 import shlex
 from operator import itemgetter
 
+from ansible.compat.six import PY3
```

Comment:
This won't work from module code because we aren't guaranteed that ansible is installed on the remote machine.  Only things inside of ```ansible.module_utils``` are transparently included in what we send over the wire with the module.  You could use ```from ansible.module_utils.six import PY3``` but there is a better way.  In the imports, use:

``` python
from ansible.module_utils._text import to_native
```